### PR TITLE
Add Code Of Conduct

### PR DESCRIPTION
**Pull request by ivannieto**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Adding Code Of Conduct file to the project root

### CODE REVIEW

**Comment by Hrxn** [comment]
Why?

**Comment by ivannieto** [comment]
Because there is no Code Of Conduct in this repo and it's a good feature to be added and to be known by all the maintainers and contributors.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (yan12125) [inline CODE_OF_CONDUCT.md]: Need an email

**Comment by Hrxn** [comment]
You marked this PR with 'Improvement'.

Why is adding a Code of Conduct an improvement?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (ivannieto) [inline CODE_OF_CONDUCT.md]: Yep, sorry. I can't find any e-mail adress in the project. Can you tell me where to find it?

**Comment by ivannieto** [comment]
@Hrxn the content of the document itself it's pretty self-explanatory.

It states some guidelines on how to behave into an Open Source project. This is an improvement.  If there were a checkbox to add docs, it should be marked as docs, but it doesn't exists. Is it a new extractor? Is it a new bug? Is it a new feature? Does it change the code? No, no, no and no. 

;)

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (yan12125) [inline CODE_OF_CONDUCT.md]: Each maintainer has his/her own email, but there's no "email for the project".

Offensive actions are rare in youtube-dl. In previous cases, either comments are deleted or an issue is locked. All happens on Github so there's not an email account for this.

**Comment by dstftw** [comment]
This is exact copy from http://contributor-covenant.org/version/1/4/code_of_conduct.md. Even though it's accepted why not just link it?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (ivannieto) [inline CODE_OF_CONDUCT.md]: It's not because of their rarity. It's made to avoid those situations. You're free to accept it.

Comment (yan12125) [inline CODE_OF_CONDUCT.md]: Yes rare or not does not matter. What I want to say is that wordings here need modifications.

**Comment by yan12125** [comment]
I'd like to say something about this pull request.

First, it's not well formed - the template provided at contributor-covenant.org can't apply to random projects directly. It requires adding an email address. Apparently the author didn't notice this, so I guess he didn't even read the whole text of that template. I feel it not a serious PR so I hesitate to continue it.

Second, I don't think adding a code of conduct would bring any real benefit. Those who trolls continue to troll. They just troll without reading the code of conduct.

Third, I don't like the idea behind 24pullrequests. IMO when you found something in a project should be changed, you open a pull request. 24pullrequests encourages the reverse - you want to open a pull request, so you find something in a project to be changed. That's not how open source works.

**Comment by ivannieto** [comment]
@yan12125 Yep, I can say it was totally my fault. Sorry. Never intented to create some kind of controversy. ❤️

**Pull request closed without merging**